### ISSUE TITLE
[BUG] open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directory

### ISSUE THREAD

**Issue by beatyman** [openyurtio/yurt-app-manager#90]
_803 10:17:49.305182       1 leaderelection.go:248] attempting to acquire leader lease kube-system/yurt-app-manager...
2022-08-03T10:17:49.305Z	INFO	controller-runtime.manager	starting metrics server	{"path": "/metrics"}
2022-08-03T10:17:49.305Z	INFO	controller-runtime.webhook.webhooks	starting webhook server
2022-08-03T10:17:49.305Z	ERROR	controller-runtime.manager	error received after stop sequence was engaged	{"error": "context canceled"}
2022-08-03T10:17:49.305Z	ERROR	controller-runtime.manager	error received after stop sequence was engaged	{"error": "context canceled"}
E0803 10:17:49.305737       1 leaderelection.go:330] error retrieving resource lock kube-system/yurt-app-manager: Get "https://10.10.0.1:443/apis/coordination.k8s.io/v1/namespaces/kube-system/leases/yurt-app-manager": context canceled
**2022-08-03T10:17:49.305Z	ERROR	setup	problem running manager	{"error": "open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directory"}**
github.com/openyurtio/yurt-app-manager/cmd/yurt-app-manager/app.Run
	/build/cmd/yurt-app-manager/app/core.go:167
github.com/openyurtio/yurt-app-manager/cmd/yurt-app-manager/app.NewCmdYurtAppManager.func1
	/build/cmd/yurt-app-manager/app/core.go:87
github.com/spf13/cobra.(*Command).execute
	/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856
github.com/spf13/cobra.(*Command).ExecuteC
	/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960
github.com/spf13/cobra.(*Command).Execute
	/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main
	/build/cmd/yurt-app-manager/app-manager.go:37
runtime.main
	/usr/local/go/src/runtime/proc.go:255
_

**Comment by beatyman** [comment]
![企业微信截图_16595220718207](https://user-images.githubusercontent.com/16757974/182585439-0a911009-2685-4b93-8507-d537c7cf475c.png)

**Comment by huiwq1990** [comment]
Try the command below:

```shell
IMAGE_TAG=v0.7.0-beta.1 TARGET_PLATFORMS=linux/amd64 make docker-build

helm uninstall yurt-app-manager  
helm install yurt-app-manager ./charts/yurt-app-manager/  
```

**Comment by Rudolf1998** [comment]
> _803 10:17:49.305182 1 leaderelection.go:248] attempting to acquire leader lease kube-system/yurt-app-manager... 2022-08-03T10:17:49.305Z INFO controller-runtime.manager starting metrics server {"path": "/metrics"} 2022-08-03T10:17:49.305Z INFO controller-runtime.webhook.webhooks starting webhook server 2022-08-03T10:17:49.305Z ERROR controller-runtime.manager error received after stop sequence was engaged {"error": "context canceled"} 2022-08-03T10:17:49.305Z ERROR controller-runtime.manager error received after stop sequence was engaged {"error": "context canceled"} E0803 10:17:49.305737 1 leaderelection.go:330] error retrieving resource lock kube-system/yurt-app-manager: Get "https://10.10.0.1:443/apis/coordination.k8s.io/v1/namespaces/kube-system/leases/yurt-app-manager": context canceled **2022-08-03T10:17:49.305Z ERROR setup problem running manager {"error": "open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directory"}** github.com/openyurtio/yurt-app-manager/cmd/yurt-app-manager/app.Run /build/cmd/yurt-app-manager/app/core.go:167 github.com/openyurtio/yurt-app-manager/cmd/yurt-app-manager/app.NewCmdYurtAppManager.func1 /build/cmd/yurt-app-manager/app/core.go:87 github.com/spf13/cobra.(*Command).execute /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 github.com/spf13/cobra.(*Command).ExecuteC /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 github.com/spf13/cobra.(*Command).Execute /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897 main.main /build/cmd/yurt-app-manager/app-manager.go:37 runtime.main /usr/local/go/src/runtime/proc.go:255 _

Could you show me the steps of your operation and the version of your environment

**Comment by AndyEWang** [comment]
same in the latest image openyurt/yurt-app-manager:latest
I found the secret is empty in config/setup/all_in_one.yaml
`
      volumes:
      - name: cert
        secret:
          defaultMode: 420
          secretName: yurt-app-manager


# Source: yurt-app-manager/templates/secret.yaml
apiVersion: v1
kind: Secret
metadata:
  name: yurt-app-manager
  namespace: kube-system
`
So we need to manually add the cert firstly, right?

**Comment by rambohe-ch** [comment]
@huiwq1990 I think we need to update `config/setup/all_in_one.yaml` file in terms of helm charts of yurt-app-manager.

**Comment by Lulucyliu** [comment]
I have the same problem

**Comment by rambohe-ch** [comment]
@beatyman The problem has been solved by #101 . please you can take a try with the newest all_in_one.yaml, and i will close this issue at first.
